### ISSUE TITLE
MinMinGuard conflicts with Samsung's Private Mode

### ISSUE THREAD

**Issue by cs-usst** [chiehmin/MinMinGuard#163]
MinMinGuard is more powerful and smarter than any other adblockers. But the problem I encounter with is I can't enable Samsung's Private Mode (on Galaxy S7e) when MinMinGuard is installed. It seems that it's something about "secure storage", but I'm not sure.
Is there anything available for the conflict?
(I'm pretty sure MinMinGuard caused this. I did a pure system installation and installed minminguard, the private mode is unavailable. Then I uninstall it, it's back again.)

**Comment by Menchen** [comment]
It's caused by xposed, you need a patch to enable private mode, try to install xSecureStorage in xposed.

**Comment by chiehmin** [comment]
@Menchen Thanks for explaining. Close this issue.

**Comment by cs-usst** [comment]
I installed the module and it didn't work.
And I found that the Private Mode works only if i shut down my phone unexpectedly (e.g. run out of power) or I use the Reboot option in Xposed Installer to restart my phone. And the Soft Reboot option won't bring up Private Mode.
btw i tried to do a clean installation and installed Magisk and Xposed and their modules one by one and see when will Private Mode stop working. And my conclusion is it stops after i install MinMinGuard and it works again when i uninstall it.
So I think the reason is more complicated.
